main.cpp

```diff
@@ -14,7 +14,50 @@ Create a branch named Part2
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Person
+{
+  int age;
+  int height;
+  float hairLength;
+  float GPA;
+  unsigned int SATScore;
+  int distanceTraveled;
+
+  void run(int howFast, bool startWithLeftFoot);
+
+  struct Foot
+  {
+      int numberOfSteps = 0;
+      int step = 5;
+
+      void stepForward()
+      {
+        numberOfSteps += 1;
+      }
+      int stepSize(int speed)
+      {
+        return speed * step;
```

Comment:
Good, but fix the formatting.   the function bodies are not indented correctly
- [ ] fix the formatting.